components/detail/SwapDetailSummary/SwapDetailAddress.tsx

```diff
@@ -18,6 +23,27 @@ function SwapDetailAddress(props: SwapDetailItem) {
         {wallet?.address}
       </Link>
       <ButtonCopyIcon tooltipText="Copy Address" text={wallet?.address} />
+      <button
+        className={`ml-1.5 flex items-center relative`}
+        onClick={() =>
+          router.push({
+            pathname: '/search',
+            query: { query: wallet?.address },
+          })
+        }>
+        <Image
```

Comment:
Rango logo width and height are 18 px in Figma.

NIT: I think it is better to enter a number for the length and width directly.
Like this:

w-[18px] h-[18px]

Reply:
fixed